drivers/volume/portworx/portworx.go

```diff
@@ -952,6 +995,64 @@ func (d *portworx) WaitDriverDownOnNode(n node.Node) error {
 	return nil
 }
 
+// ValidateStoragePoolsSizes checks if sizes of storage pool are expanded based on autopilot rules
+func (d *portworx) ValidateStoragePoolsSizes(apRules []apapi.AutopilotRule) error {
```

Comment:
No need to pass in apRules.

The portworx volume driver has an instance of k8s sched-ops (https://github.com/portworx/torpedo/tree/master/drivers/volume/portworx/schedops) and it can simply use that to enumerate all autopilot rules that affect the storage pools being validated.

This is a better check anyways since the volume driver validate does not depend on it being provided a correct list of rules but instead it will independently figure out the list of apRules that affect the pool.